### PR TITLE
Extend Adwaita workaround [#2305]

### LINKED ISSUE #2305: Ubuntu 18.10 theme mismatch, input and title boxes white
How it looks in 18.04:
image

How it looks in 18.10:
image

Note the title/user bar at the top and the input box at the bottom are both white now.

I recompiled with source just to get to the theme manager, which made no difference. Open to ideas, but GUI work is completely out of my wheelhouse. #1832 offers a temporary solution that makes the input box text legible, but I'm assuming another similar fix needs to be made for the new version of GNOME?

### PR DESCRIPTION
**Pull request by cajuncooks**
Resolves #2305.

### CODE REVIEW

File: src/fe-gtk/fe-gtk.c [hunk 1]
```diff
@@ -293,10 +293,10 @@ create_input_style (GtkStyle *style)
 		GtkSettings *settings = gtk_settings_get_default ();
 		char *theme_name;
 
-		/* gnome-themes-standard 3.20 relies on images to do theming
+		/* gnome-themes-standard 3.20+ relies on images to do theming
 		 * so we have to override that. */
 		g_object_get (settings, "gtk-theme-name", &theme_name, NULL);
-		if (g_str_has_prefix (theme_name, "Adwaita"))
```
Comment (TingPing) [inline src/fe-gtk/fe-gtk.c]: Can you test Yaru-dark also and change it to a prefix check like Adwaita?

**Comment by TingPing** [comment]
Also nitpick but I'd prefer the commit message:

```
Extend input box GTK theme workaround to include Yaru

Fixes #2305
```

File: src/fe-gtk/fe-gtk.c (hunk 1 above)
Comment (cajuncooks) [reply src/fe-gtk/fe-gtk.c]: I ran `gsettings set org.gnome.desktop.interface gtk-theme Yaru-dark` and confirmed application response (Chromium turned dark, text boxes in question did too, but lost their theme). Changing to a prefix check resolves this, and maintains the theme through a gtk-theme update while Hexchat is open. I've also updated the commit message.

**Comment by TingPing** [comment]
Thanks!

**Pull request merged**